cv_next/app/cv/[cvId]/components/commentSection/commentForm.tsx

```diff
@@ -14,15 +14,15 @@ export default function CommentForm({ cv }: { cv: CvModel }) {
   const supabase = useSupabase();
 
   const formAction = async (formData: FormData) => {
-    checkValidityOfComment()
-    const userId = (await supabase.auth.getUser()).data.user?.id
-    if (!userId) throw new Error("User not found") // TODO: handle this
-    const comment: NewCommentModel = {
-      data: formData.get("comment") as string,
-      document_id: cv.id,
-      parent_comment_Id: null,
-      user_id: userId,
-    }
+    const userId = (await supabase.auth.getUser()).data.user?.id;
+    if (!userId) router.push("/inactive");
```

Comment:
Same here like the comment on `comment.tsx`.

Reply:
I uploaded a fix for that in the SSR branch as this one doesn;t have the next page param